dart/test/flat_buffers_test.dart

```diff
@@ -666,6 +666,7 @@ class BuilderTest {
     // read and verify
     BufferContext buf = BufferContext.fromBytes(byteList);
     List<int> items = const Uint16ListReader().read(buf, 0);
+    expect(items is Uint16List, true, skip: Endian.host == Endian.big);
```

Comment:
@vaind I changed this test to skip this assert on `Endian.big`. The documentation for skip suggests `If [skip] is a String or true, the assertion is skipped. The arguments are still evaluated, but [actual] is not verified to match [matcher]. If [actual] is a [Future], the test won't complete until the future emits a value.`, so it should be fine. I didn't see any cases where we conditionally perform an assert in tests, so if there is an expected pattern between using skip or just wrapping it with an if condition, let me know.

Reply:
skip is fine, or you can do the following:
```suggestion
    expect(items is Uint16List, Endian.host == Endian.little);
```

Reply:
I like that better. Thanks.